eve/workers/end2end/scripts/deploy-zenko.sh

```diff
@@ -65,11 +65,11 @@ spec:
   versions:
     management:
```

Comment:
Do we need to add vault in this file as well?
I do not know how this works yet. But still checking

Reply:
Vault is optional and has to be enabled which is not the case in the zenko CR below. Now if you think we should be using it here that is a different question that I am not really the one to answer :|

Reply:
I might not have the complete context, clarifying just to learn more.
Are you saying Vault is optional in Zenko 2.0.0-alpha.X?
or 
Vault is optional in end2end tests?

Reply:
> Are you saying Vault is optional in Zenko 2.0.0-alpha.X?

Currently yes, it must be enabled through the zenko CR.

> Vault is optional in end2end tests?

Also yes as currently, it seems to be disabled.